studio/lib/helpers.ts

```diff
@@ -165,39 +165,57 @@ export const copyToClipboard = (str: string, callback = () => {}) => {
   }
 }
 
-export async function passwordStrength(value: string) {
+type PasswordStrengthResponse = {
+  message: string
+  warning: string | null
+  strength: number
+}
+export async function passwordStrength(value: string): Promise<PasswordStrengthResponse> {
   let message = ''
   let warning = ''
   let strength = 0
 
-  if (value && value !== '') {
-    if (value.length < 99){
-      const response = await post(`${API_URL}/profile/password-check`, { password: value })
-      if (!response.error) {
-        const { result } = response
-        const score = (PASSWORD_STRENGTH as any)[result.score]
-        const suggestions = result.feedback?.suggestions ? result.feedback.suggestions.join(' ') : ''
-
-        // set message :string
-        message = `${score} ${suggestions}`
-
-        // set strength :number
-        strength = result.score
-
-        // warning message for anything below 4 strength :string
-        if (result.score < DEFAULT_MINIMUM_PASSWORD_STRENGTH) {
-          warning = `${
-            result?.feedback?.warning ? result?.feedback?.warning + '.' : ''
-          } You need a stronger password.`
-        }
-      }
-    } else {
-      message = `${PASSWORD_STRENGTH[0]} Passwords must be 99 or fewer characters.`
-      strength = 0;
-      warning = 'You need a shorter password.'
+  if (!value) {
+    return {
+      message,
+      warning,
+      strength,
     }
   }
 
+  if (value.length >= 100) {
```

Comment:
```suggestion
  if (value.length <= 99) {
```
or
```suggestion
  if (value.length < 100) {
```

Currently the password has to be greater than or equal to 100, but want it to be less than or equal to 99.

Reply:
wait never mind this is actually correct.